addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -160,10 +160,17 @@ def res_partner_create(self, email, name, company):
         return response
 
     @http.route('/mail_plugin/log_mail_content', type="json", auth="outlook", cors="*")
-    def log_mail_content(self, model, res_id, message):
+    def log_mail_content(self, model, res_id, message, attachments=None):
```

Comment:
Would be nice to explain the expected structure of this new parameter in the docstring.

Also the limitations that apply to it in terms of size etc?

Because in your commit message you detail various limitations but I don't see anything in the code here to enforce them?
I'm also noticing that the commit message is a copy/paste of the one from the Gmail plugin PR so maybe that's why.
Maybe you should differentiate them a bit so that it's less confusing as to what side handles what.

Reply:
The limitation is only on the Gmail side, I'll update the commit message